examples/depthEstimation-single-image/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>ml5.js | Depth Estimation Single Image</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.9.4/p5.min.js"></script>
+    <script src="../../dist/ml5.js"></script>
+  </head>
+
+  <body>
+    <script src="sketch.js"></script>
+  </body>
+</html>
```

Comment:
Since git is operating on entire lines, it expects the last line of code to still be followed by a newline character - that's what this warning is about. (Just pressing enter one more time here and in some other files should address this.)